src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -4175,7 +4175,7 @@ impl Coordinator {
                     future: batch,
                     span: _,
                 } => {
-                    // TODO: This timeout should be removed once #11782 lands;
+                    // TODO(jkosh44): This timeout should be removed;
```

Comment:
@jkosh44 I'm not sure how to fix that, please follow up on it, #11782 has landed.

Reply:
Ack. This seems like a non-trivial fix, so I think you did the right thing of leaving the TODO but removing the reference to #11782.